python/replicate/system.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+
+def get_python_version():
+    """
+    Returns the Python version of the experiment as a str.
+    """
+    return sys.version.split(" ")[0]
```

Comment:
A more robust implementation might be to use `sys.version_info` tuple and join the first 3 elements.